config.example.yml

```diff
@@ -6,3 +6,7 @@ token: mfa.exampletoken
 
 # The command prefix used by the bot.
 command_prefix: '!!'
+
+# Cogs to load.
+cogs:
+  - union.cogs.utility
```

Comment:
This is very minor, but we probably want to standardize line-endings and file-ending newlines in the contributing.md as well.